_grab_ui.py

```diff
@@ -641,6 +502,8 @@ def on_key(self, key_event: KeyEvent) -> None:
 
     def perform_action(self, action: Tuple[ActionName, ActionArgs]) -> None:
         func, args = action
+        if len(args) == 1:  # TODO how to do this correctly?
```

Comment:
I expect this will be fixed when parser functions decorated with `@func_with_args` start returning proper tuples.

Reply:
Indeed :)